CONTRIBUTING.md

```diff
@@ -26,133 +26,169 @@ We specifically do not merge PRs **without prior issues** that:
+1. You should write most, if not all of your code yourself.
+2. Do not copy code from other authors without permission. If you have permission, always add the `Co-authored-by` sign-off to your commits.
+3. When you copy/paste code from within the OpenTofu code, always make it explicit where you copied from.
+4. When you copy from external sources, always make explicit where your copy is coming from and make sure that their license and copyright attribution requirements are met.
+5. Specifically, do not copy from the Terraform repository, or any PRs filed against that repository. Doing so will immediately disqualify your PR from being merged.
 
-If not using the devcontainer (if using an incompatible IDE), you can still still build with Docker (and thus need not install any dependencies locally) by running docker commands directly. See the [Building with Docker](#building-with-docker) section.
+> [!WARNING]
+> Coding assistants based on large language models (LLMs), such as ChatGPT or GitHub Copilot, are awesome tools to help. However, in the specific case of OpenTofu the training data may include the BSL-licensed Terraform. Since the OpenTofu/Terraform codebase is very specific and LLMs don't have any other training source, they may emit copyrighted code. Please avoid using LLM-based coding assistants as much as possible. If your PR is found to contain code matching Terraform, your PR will be rejected, and you will not be permitted to work on that feature.
 
-### Building Natively
+---
 
-Refer to the file [`.go-version`](.go-version) to see which version of Go OpenTofu is currently built with. Other versions will often work, but if you run into any build or testing problems please try with the specific Go version indicated. You can optionally simplify the installation of multiple specific versions of Go on your system by installing [`goenv`](https://github.com/syndbg/goenv), which reads `.go-version` and automatically selects the correct Go version.
+## Advanced topics
 
-### Build with Go
+### Acceptance Tests: Testing interactions with external services
 
-Switch into the root directory of the cloned repository and build OpenTofu using the Go toolchain in the standard way:
+The test command above only runs the self-contained tests that run without external services. There are, however, some optional tests in the OpenTofu CLI codebase that *do* interact with external services. We collectively refer to them as "acceptance tests".
 
-```sh
-cd opentofu
-go install ./cmd/tofu
```

Comment:
I wonder why you decided to change this explanation and use the `go build` instead? What was the reason?

Reply:
Because it's not in line with the containerized description below and it's also not intuitive that this will build and install the binaries.